src/main/java/com/microfocus/application/automation/tools/run/RunFromFileBuilder.java

```diff
@@ -286,6 +293,61 @@ public void setFsTests(String fsTests) {
         runFromFileModel.setFsTests(fsTests);
     }
 
+    public String getNumberOfReruns() { return runFromFileModel.getNumberOfReruns(); }
```

Comment:
When you will introduce your own model that will implement abstractDescriableImpl there won't be a need in exposing the field getters/setters.